site/website/blog/2020-07-16-designing-evergreen-cache-cdc.md

```diff
@@ -270,28 +268,26 @@ The pipeline definition is quite straightforward:
 ```java
 pipeline.readFrom(source)                                       //1
         .withoutTimestamps()
-        .map(r -> {
-            Person person = r.value().toObject(Person.class);   //2
-            return Util.entry(person.id, person);               //3
-        })
-        .writeTo(Sinks.remoteMap(                               //4
-                "entities",                                     //5
-                new CustomClientConfig(env.get("CACHE_HOST"))   //6
+        .writeTo(CdcSinks.remoteMap(                            //2
```

Comment:
This API is more compact than the previous one, but it's more complex to get the hang on when you're unfamiliar with it.

Do you intend to freeze the API at some point?

Reply:
It's not a matter of API. There was no API change, the pipeline simply wasn't using the right sink. This sink is the right one because it handles deletions too. The old one wasn't.